titapapunne/MoreThanWords
# #65 : desc in app.py and fetch in HomeScreen.js

**app.py**
- Added to the outcomes endpoint in app.py to calculate the total number of outcomes and then used that info in the frontend of HomeScreen.js for the progress bars; should still work if new outcomes are added
- Use desc and "Outcomes__c" to filter related field
- transform json data to python dictionary(easier to process data without hardcoding)
- process data to get the sum for all outcomes in the Trainee pod

**HomeScreen.js**
- Change to using real user data instead of fake user data by passing in the user parameter in app.py and by using token in HomeScreen.js (also changed from axios to fetch).


-----------------------------------------------------------------------------------
Steps to test PR: 
Login in on app using Yichen's account to test having real user data:
> Username: my.waymao@gmail.com
> password: ZijReFQ@AJYz3jfrA3iz

Then the Home Screen should load all the progress bars with the correct number of total outcomes.
> ( Trainee : 3, Associate: 0, Partner: 0)
-----------------------------------------------------------------------------------

Standard Checklist:
